packages/design-system/src/components/Table/Table.jsx

```diff
@@ -102,17 +126,24 @@ Table.propTypes = {
    */
   className: PropTypes.string,
   /**
-   * Apply vertically stacked row style at different viewpoint sizes.
+   * Applies responsive styles to vertically stacked rows at different viewpoint sizes.
+   * Required props to be used in conjunction with this includes `headers`, `id` and `stackedTitle`
```

Comment:
Slight wording suggestions. Also I know I originally suggested this wording haha
"Applies responsive, stacked table style at the specified breakpoint. When `stackBreakpoint` is set, `headers`, `id` and `stackedTitle` props are required in `TableHeaderCell` and `TableDataCell`.

Reply:
Also is that how it works? Are all 3 props required for both header cell and data cell? we should be clear which props are required where.